deepchem/data/data_loader.py

```diff
@@ -1613,3 +1616,23 @@ def _featurize_shard(  # type: ignore[override]
             ids.append(entry_id)
         X = np.concatenate(features, axis=0)
         return X, np.array(labels), np.array(weights), np.array(ids)
+
+class DFTYamlLoader(DataLoader):
```

Comment:
Can you add the usage example and the docs?

Reply:
Add a full description of the expected file format for the YAML in this docstring. Remember that this is the main documentation for this filetype so it should have an example of the expected format and ideally references to other papers that use this format